kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -694,19 +693,22 @@
                   ...this.$route,
                   params: {
                     ...this.$route.params,
-                    subtopic: t.id,
+                    subtopic: c.id,
                   },
                 }
                 : null;
-
-            return {
-              ...t,
-              viewAll,
-              children,
-              showMore,
-              viewMore,
-            };
-          });
+            // Having processed the topic, just add it to the end of the array.
```

Comment:
> the end of the array
Is this the... first level of the array? the array that represents the current view with each sub-array (or topic) being one row of cards?

Reply:
The overall array is all the content nodes for the current topic - with the resources clumped together into sub arrays as described above.